kombu/async/http/__init__.py

```diff
@@ -17,7 +18,16 @@ def get_client(hub=None, **kwargs):
     """Get or create HTTP client bound to the current event loop."""
     hub = hub or get_event_loop()
     try:
+        if not hub:
+            hub = Hub()
+            client = hub._current_http_client = Client(hub, **kwargs)
+            set_event_loop(hub)
+
         return hub._current_http_client
     except AttributeError:
+        if not hub:
+            hub = Hub()
+
         client = hub._current_http_client = Client(hub, **kwargs)
```

Comment:
It seems that this line would actually set the `_current_http_client` attribute, as it was before. Could you please provide some details on the changes?